core/state_processor_test.go

```diff
@@ -616,3 +625,902 @@ func getParentBlockHash(statedb *state.StateDB, number uint64) common.Hash {
+	ChainID:                 big.NewInt(69420),
+	HomesteadBlock:          big.NewInt(0),
+	EIP150Block:             big.NewInt(0),
+	EIP155Block:             big.NewInt(0),
+	EIP158Block:             big.NewInt(0),
+	ByzantiumBlock:          big.NewInt(0),
+	ConstantinopleBlock:     big.NewInt(0),
+	PetersburgBlock:         big.NewInt(0),
+	IstanbulBlock:           big.NewInt(0),
+	MuirGlacierBlock:        big.NewInt(0),
+	BerlinBlock:             big.NewInt(0),
+	LondonBlock:             big.NewInt(0),
+	Ethash:                  new(params.EthashConfig),
+	ShanghaiTime:            u64(0),
+	VerkleTime:              u64(0),
+	TerminalTotalDifficulty: common.Big0,
+}
+
+// TestProcessVerkleInvalidContractCreation checks for several modes of contract creation failures
+func TestProcessVerkleInvalidContractCreation(t *testing.T) {
+	var (
+		coinbase = common.HexToAddress("0x71562b71999873DB5b286dF957af199Ec94617F7")
+		account1 = common.HexToAddress("0x687704DB07e902e9A8B3754031D168D46E3D586e")
+		account2 = common.HexToAddress("0x6177843db3138ae69679A54b95cf345ED759450d")
+		gspec    = &Genesis{
+			Config: testKaustinenLikeChainConfig,
+			Alloc: GenesisAlloc{
+				coinbase: {
+					Balance: big.NewInt(1000000000000000000), // 1 ether
+					Nonce:   0,
+				},
+				account1: {
+					Balance: big.NewInt(1000000000000000000), // 1 ether
+					Nonce:   0,
+				},
+				account2: {
+					Balance: big.NewInt(1000000000000000000), // 1 ether
+					Nonce:   1,
+				},
+				params.BeaconRootsAddress:        {Nonce: 1, Code: params.BeaconRootsCode, Balance: common.Big0},
+				params.HistoryStorageAddress:     {Nonce: 1, Code: params.HistoryStorageCode, Balance: common.Big0},
+				params.WithdrawalQueueAddress:    {Nonce: 1, Code: params.WithdrawalQueueCode, Balance: common.Big0},
+				params.ConsolidationQueueAddress: {Nonce: 1, Code: params.ConsolidationQueueCode, Balance: common.Big0},
+			},
+		}
+	)
+	// Create two blocks that reproduce what is happening on kaustinen.
+	// - The first block contains two failing contract creation transactions, that
+	//   write to storage before they revert.
+	//
+	// - The second block contains a single failing contract creation transaction,
+	//   that fails right off the bat.
+	_, chain, _, _, statediffs := GenerateVerkleChainWithGenesis(gspec, beacon.New(ethash.NewFaker()), 2, func(i int, gen *BlockGen) {
+		gen.SetPoS()
+
+		if i == 0 {
+			var tx1, tx2, tx3 types.Transaction
+			// SSTORE at slot 41 and reverts
+			tx1payload := common.Hex2Bytes("f8d48084479c2c18830186a08080b8806000602955bda3f9600060ca55600060695523b360006039551983576000601255b0620c2fde2c592ac2600060bc55e0ac6000606455a63e22600060e655eb607e605c5360a2605d5360c7605e53601d605f5360eb606053606b606153608e60625360816063536079606453601e60655360fc60665360b7606753608b60685383021e7ca0cc20c65a97d2e526b8ec0f4266e8b01bdcde43b9aeb59d8bfb44e8eb8119c109a07a8e751813ae1b2ce734960dbc39a4f954917d7822a2c5d1dca18b06c584131f")
```

Comment:
"SSTORE at slot 41 and reverts" ???
No. It starts with `f8`, which is the eof `EXTCALL` opcode, for starters. The one below, "SSTORE at slot 133 and reverts" starts with `02`, a.k.a `MUL`, so obviously croaks on shallow stack immediately. 

This kind of highlights the unmaintainability of binary blobs like this. IF you need particular code, either 

1. Use slices of vm.Opcode (for simple stuff). 
2. Use assembly programs, and feed them through some asm -> opcodes engine.

Tangential: What is the point of the payloads in all of these transactions?

Reply:
these payloads aren't EOF, they are raw transactions extracted from the testnet. They are blobs because they are extracted from a real life testnet block.